### PR TITLE
Back to profile button in Edit profile page

### PR DESCRIPTION
**Pull request by carloshfoliveira**
Added a button that takes the user back to its profile general view.

Signed-off-by: Carlos Oliveira <carlospecter@gmail.com>
Signed-off-by: Peter Lima <peterlima@gmail.com>

### CODE REVIEW

File: colab/static/css/screen.css [hunk 1]
```diff
@@ -115,10 +115,9 @@ form.signup .form-group {
     margin-left: 0.5em;
 }
 
-div.submit {
```
Comment (alexandreab) [inline colab/static/css/screen.css]: This class is already used in:
```
colab/accounts/templates/accounts/user_create_form.html
colab/accounts/templates/registration/password_reset_confirm_custom.html
colab/accounts/templates/registration/password_change_form_custom.html
colab/accounts/templates/registration/login.html
colab/accounts/templates/registration/password_reset_form_custom.html
```

**Comment by carloshfoliveira** [comment]
I've return its definition, since it doesn't conflicts with the new div that groups update and back buttons, it works like a charm.

**Comment by carloshfoliveira** [comment]
login
password_reset
signup_form
update_profile

**Comment by MatheusFaria** [comment]
The translation pt-br for the "Go to profile panel" is missing

**Pull request merged**